README.md

```diff
@@ -7,3 +7,11 @@ A ratatui widget for drawing a plotters chart.
 <img src="examples/mandelbrot.png" width=200px />
 <img src="examples/normal-dist.png" width=200px />
 <img src="examples/3d-plot.png" width=200px />
+
+## Tests
+
+When running tests please note the following constraints.
+
+The tests *require* human interaction.  An image is displayed in the upper section of the terminal, with a yes/no question in the lower section.  Inspect the image, and respond to the question with 'y' or 'n' as appropriate.
```

Comment:
Hmm, these are called examples not tests.

Reply:
You may want to look at how ratatui tests its own widgets,

Reply:
I humbly disagree - these are tests not examples.  I didn't like them needing a human to perform the pass/fail assessment, but I found them more helpful than having no tests :)  

Thank you for the pointer to Ratatui's test capabilities.  I'll rework work and resubmit at some point.